### DISCUSSION TITLE
What is the difference between Query text and Refinement filters

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#1557] [General]
I added the PnP Modern Search web part from this link @ https://github.com/microsoft-search/pnp-modern-search/releases . and inside the "PnP - Search Results" web=part, i have 2 sections; Query Text & Refinement filters, as follow:-

![152339-2filters](https://user-images.githubusercontent.com/79726511/143420892-f42658ad-e142-47a7-b868-94f955862bb7.png)

So what is the difference between these 2 options? and when to use each?

Thanks

**Comment by wobba** [comment]
See https://microsoft-search.github.io/pnp-modern-search/usage/search-results/data-sources/ (and KQL is not supported - docs will be updated). It's pretty much similar to adding the same expression to the query template. @FranckyC if you have a filter web part, should it pre-select manually added filters, if the value match? Or is it merely a FQL field?
